### DISCUSSION TITLE
Export chart to JPG/PDF...

### DISCUSSION THREAD

**Discussion by StephaneSynergie** [Live-Charts/LiveCharts2#820] [Q&A] [answered]
Hello, thx for this library, easy to use, easy to understand and with well render :)

Is there a native way to export/print charts to any standard format as pdf, jpg, bitmap, tiff...

Regards :) 

Steph

**Comment by beto-rodriguez** [comment, marked as answer]
Hi! glad to hear that the library is easy to use, that is the idea!

Yes, you can render images, take a look at:
https://lvcharts.com/docs/WPF/2.0.0-beta.330/samples.general.chartToImage

For PDF:
https://github.com/beto-rodriguez/LiveCharts2/blob/master/samples/QuestPDFSample/Program.cs

**Comment by StephaneSynergie** [comment]
thx for the quick reply, seems i've looked in the docs with both eyes closed :D
